src/pages/Integrations/Create/schema.ts

```diff
@@ -72,8 +83,12 @@ export const schema = (category) => ({
                     name: 'secret-token',
                     type: 'text',
                     label: 'Secret token',
-                    helperText:
-                      'The defined secret token is sent as a "X-Insight-Token" header on the request.',
+                    helperText: IntegrationType.SERVICE_NOW
```

Comment:
I double checked and this piece isn't working. @InsaneZein any chance you could check it out? I feel like its pretty straight forward but its only showing the ServiceNow text.